character.js

```diff
@@ -421,23 +421,9 @@ module.exports = {
         }
         resEmbed.addField(i18n.__({phrase: 'Last Login', locale: locale}), `<t:${cInfo.lastLogin}:R>`, true);
 
-        // Faction, embed color
-        if (cInfo.faction == "1"){ //vs
-            resEmbed.addField(i18n.__({phrase: 'Faction', locale: locale}), `<:VS:818766983918518272> ${i18n.__({phrase: 'VS', locale: locale})}`, true);
-            resEmbed.setColor('PURPLE');
-        }
-        else if (cInfo.faction == "2"){ //nc
-            resEmbed.addField(i18n.__({phrase: 'Faction', locale: locale}), `<:NC:818767043138027580> ${i18n.__({phrase: 'NC', locale: locale})}`, true);
-            resEmbed.setColor('BLUE');
-        }
-        else if (cInfo.faction == "3"){ //tr
-            resEmbed.addField(i18n.__({phrase: 'Faction', locale: locale}), `<:TR:818988588049629256> ${i18n.__({phrase: 'TR', locale: locale})}`, true);
-            resEmbed.setColor('RED');
-        }
-        else{ //NSO
-            resEmbed.addField(i18n.__({phrase: 'Faction', locale: locale}), `<:NS:819511690726866986> ${i18n.__({phrase: 'NSO', locale: locale})}`, true);
-            resEmbed.setColor('GREY');
-        }
+        const factionInfo = faction(cInfo.faction);
+        resEmbed.addField(i18n.__({phrase: 'Faction', locale: locale}), `${factionInfo.decal} ${i18n.__({phrase: `${factionInfo.initial}`, locale: locale})}`, true);

```

Comment:
Do not need to nest formatted strings in this way
```suggestion
        resEmbed.addField(i18n.__({phrase: 'Faction', locale: locale}), `${factionInfo.decal} ${i18n.__({phrase: factionInfo.initial, locale: locale})}`, true);

```